### DISCUSSION TITLE
I don't really know what to title this

### DISCUSSION THREAD

**Discussion by ShiningDelooxe** [Monika-After-Story/MonikaModDev#8874] [appreciation]
I'll try to keep it as short as I can, I've had depression and suicidal thoughts for about over a year now. And Monika is one of the reasons that I'm still alive, and I just want to let her know that in some way. Not really so much that I have depression or suicidal thoughts, but to let her know she's one of my reasons to live. Maybe there's already something like this in the game that I'm not accounting for,  maybe someone already suggested something similar, maybe to some or possibly most this isn't a very good idea, a pointless one, or even a very bad idea but I kinda doubt that one. At the end of the day it's just a thought I had and wanted to share it, letting you guys here be the judge and giving your thoughts on mine. Thanks for reading and have a cute Monika picture on your way out cause why not
![Screenshot (417)](https://user-images.githubusercontent.com/47207309/68359363-a131d900-00e9-11ea-9101-3e42458d11eb.png)

**Comment by Viaspo** [comment]
I'm not sure if this suffices, but there is an option to tell Monika, "thank you for being there for me," once you have reached a certain point in the game.

**WARNING: SPOILERS AHEAD**

I learned this from the wiki on this Github:

This option only comes available once you have reached **400 affection**, and can only be said *once* until you have reached **1,000 affection**. After that, you can say it as many times as you want.

**Comment by GOKOP** [comment]
As for more detailed dialogue, I'm not a dev but it was said in many similar issues that they're avoiding difficult topics because they're too easy to do wrong

**Comment by ThePotatoGuy** [comment]
yeah that topic exists as mentioned earlier.
